client/landing/stepper/declarative-flow/internals/steps-repository/intro/videopress-intro-modal-content.tsx

```diff
@@ -0,0 +1,80 @@
+import { Button } from '@automattic/components';
+import { useTranslate } from 'i18n-calypso';
+import './videopress-intro-modal-styles.scss';
+import CheckmarkIcon from './icons/checkmark-icon';
+import { IntroModalContentProps } from './intro';
+
+const VideoPressIntroModalContent: React.FC< IntroModalContentProps > = ( { onSubmit } ) => {
+	const translate = useTranslate();
+
+	return (
+		<div className="videopress-intro-modal">
+			<h1 className="intro__title">{ translate( 'Your video site, with no hassle' ) }</h1>
+			<div className="intro__description">
+				{ translate(
+					'Create a WordPress.com site with everything you need to share your videos with the world.'
+				) }
+			</div>
+			<ul className="videopress-intro-modal__list">
+				<li>
+					<CheckmarkIcon />
+					<span>
+						{ translate( '{{a}}Videomaker{{/a}}, a premium theme optimized to display videos.', {
+							components: {
+								a: (
+									<a
+										href="https://jetpack.com/features/design/themes/showcase/videomaker/"
```

Comment:
Maybe we should use this link instead: https://wordpress.org/themes/videomaker/

Might be less confusing than linking to Jetpack? Or here maybe? https://videomakerdemo.wordpress.com/

Reply:
I think https://videomakerdemo.wordpress.com/ is better.

If you go to https://wordpress.org/themes/videomaker/ it makes you choose a site if you are already logged in. It isn't very useful which is why I chose the Jetpack link... but the theme demo makes more sense than jetpack.